ui/app/components/job-page/parts/children.js

```diff
@@ -31,9 +31,9 @@ export default class Children extends Component.extend(Sortable) {
     return this.get('job.taskGroups') || [];
   }
 
-  @computed('job.children.[]')
+  @computed('jobs.[]')
   get children() {
-    return this.get('job.children') || [];
+    return this.jobs || [];
```

Comment:
A big part of this PR is going from "Derive child jobs from the parent job's model's computed properties", to "accept the child jobs as passed in directly"